models/dims/contract_details_hist.sql

```diff
@@ -0,0 +1,9 @@
+{{ config(materialized = 'table') }}
+
+select
+    'dummy_contract_id' as contract_id
+    , 'dummy_ledger_key_hash' as ledger_key_hash
+    , current_timestamp() as start_date
+    , current_timestamp() as end_date
```

Comment:
Can we avoid using `current_timestamp()` functions to keep pipelines idempotent? It could be replaced with airflow date `'{{ dbt_airflow_macros.ts(timezone=none) }}'`

Example: 
https://github.com/stellar/stellar-dbt-public/blob/be4de943783abba9c4dbb3a8adaacca8097e30b9/models/marts/ledger_current_state/ttl_current.sql#L33C59-L33C103